### PR TITLE
GCI: Initial Code Structure

### PR DESCRIPTION
**Pull request by HITGIF**
- Update SDK Version to 27.0.1
- Add Dependencies for Retroifit, Butterknife and Icepick

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (garg-anuj) [inline .idea/gradle.xml]: Ignore all files under `.idea`

**Comment by HITGIF** [comment]
Done

**Comment by garg-anuj** [comment]
@HITGIF they have already been commited you will need to remove the too

**Comment by garg-anuj** [comment]
Thanks for work @HITGIF

**Pull request closed without merging**